### ISSUE TITLE
aspect text files with example Q identifiers

### ISSUE THREAD

**Issue by egonw** [WDscholia/scholia#1815]
The templates do not have examples Q identifiers for which the query is supposed to work. At the same time, the landing pages (the `empty` pages) for each aspect does have examples. I am thinking of extracting these into `${aspect}_qids.txt` in the same folder as the queries.

@fnielsen, what do you think? okay, or got a better idea that is easy to implement?

**Comment by fnielsen** [comment]
I am trying to understand the issue. For instance, on `https://scholia.toolforge.org/author/` Do you propose to extract the Q-ids referring to Südhof, Peltonen-Palotie, Musso, etc. and put then into a file called `author_qids.txt`?

What shoud the `author_qids.txt` be used for? Testing?

**Comment by egonw** [comment]
Yes, exactly: for testing. It was asked for here: https://twitter.com/klinovp/status/1492036327734497280

**Comment by egonw** [comment]
Which has this blog post as context: https://www.stardog.com/labs/blog/wikidata-in-stardog/
